platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -186,4 +214,81 @@ - (void)cancel;
     _mbglMapSnapshotter.reset();
 }
 
+- (NSURL*)styleURL;
```

Comment:
The trailing semicolon here is technically incorrect, but GCC and Clang are lenient about it by default because it used to be very common a decade or so ago. (If I remember correctly, older versions of Interface Builder would generate boilerplate code for you that included the semicolon.) Just an FYI; I thought you might find it interesting. The most substantive issue is the space before the asterisk. 😛